Doc/library/pprint.rst

```diff
@@ -107,24 +112,13 @@ The :mod:`pprint` module defines one class:
       >>> pp.pprint(tup)
       ('spam', ('eggs', ('lumberjack', ('knights', ('ni', ('dead', (...)))))))
 
-
-The :mod:`pprint` module also provides several shortcut functions:
-
 .. function:: pformat(object, indent=1, width=80, depth=None, *, \
                       compact=False, sort_dicts=True, underscore_numbers=False)
 
    Return the formatted representation of *object* as a string.  *indent*,
-   *width*, *depth*, *compact*, *sort_dicts* and *underscore_numbers* will be passed to the
-   :class:`PrettyPrinter` constructor as formatting parameters.
-
-   .. versionchanged:: 3.4
-      Added the *compact* parameter.
-
-   .. versionchanged:: 3.8
-      Added the *sort_dicts* parameter.
-
-   .. versionchanged:: 3.10
-      Added the *underscore_numbers* parameter.
```

Comment:
Why remove versionchanged notes?  People need to know this.

Reply:
We're removing the repetitions.

Reply:
But these are not needless repetitions: they are separate notes on separate functions.

Reply:
We removed repetition not only of the version labels but also of the definitions they relate to, and we replaced them by a reference to where those definitions appear (once).

It is easier read the docs when they are not repetitive because then you know that A is the same as B, and you don't need to do the work to compare them just to find that out.